RELEASING.md

```diff
@@ -0,0 +1,60 @@
+# Releasing Lucid UI
+
+Lucid UI is released to NPM as a public module.
+
+## Installation requirements
+
+- `node@10.x.x`: You can use `nvm` or otherwise to manage node versions.
+- `npm@>=6.0.0`: Use `npm i -g npm` to update your npm version.
+- `git`: You probably already have this.
+
+## Release requirements
+
+- An authorized corporate npm account.
+- `ssh` setup on GitHub. To check your credentials: `git remote -v`.
+- `.npmrc` file in your `\$HOME directory`. The `.npmrc` file is what tells `npm` _where_ the registry is.
+
+## Prerelease check
+
+Before updating the documentation and releasing to npm, follow these steps:
+
+1. `git pull origin master`: Ensure that `master` is clean and in sync with GitHub.
+2. `npm install`: Ensure that you have the latest dependencies.
+3. `npm run test`: Run the test suite--if you need to update the Jest snapshots: `npm run snapshot`.
+4. `npm login`: Ensure that you're logged into npm.
+
+## Update documentation
+
+After enrusing the release is production-ready, determine a suitable version number based on the types of changes in the release.
```

Comment:
Typo here. Looks like it should be 'ensuring'

Reply:
Good eye! Corrected.